DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -533,10 +561,9 @@ struct DataHeader : public BaseHeader
   DataOrigin dataOrigin;
 
   ///
-  /// need something for alignment, is there another field needed?
-  /// how about a hash?
+  /// How many split payloads in total
   ///
-  uint32_t reserved = 0;
+  SplitPayloadAmountType splitPayloadAmount;
```

Comment:
Amount might be interpreted as data size, IMO. 
How about splitPayloadParts ?

Reply:
why not